### ISSUE TITLE
PrefixAllGlobals: sniff should check for two-letter prefixes

### ISSUE THREAD

**Issue by Ipstenu** [WordPress/WordPress-Coding-Standards#1733]
## Is your feature request related to a problem?
The sniffer for PrefixAllGlobals does not check for two letter prefixes. This is a problem as there are roughly 80,000 (yes thousand) plugins, and a large number 

## Describe the solution you'd like

Prefixes should be checked for length. They should be at least 3 characters.

INVALID: function mp_save_post()

VALID: function myplugin_save_post()

## Additional context (optional)

This should be recommended by default

**Comment by mikelking** [comment]
I appreciate that you included examples of valid vs invalid. Curious how would this check be affected by the NAMESPACE attribute? 

If one were to use;

```PHP
NAMESPACE myplugin

function save_post() { ... }
```

**Comment by GaryJones** [comment]
In this case, that `save_post()` function in your example is not in the _global_ namespace (it's in the `myplugin` namespace), so the sniff wouldn't check it for the presence of a prefix anyway.

**Comment by mikelking** [comment]
@GaryJones 

That is what I thought. Thanks for the clarification.
